gnosis/safe-react-apps
# feat(wallet-connect): Check if app exists as a Safe App

## What it solves
resolves #167 

## How this PR fixes it
We are checking if an app exists as a Safe App. In this case we are asking the user for decide to use the Safe App or continue using Wallet Connect

As part of this task we are revamping the whole UI following this designs and workflow

https://www.figma.com/file/iicHRaXd0ypyttJqq2WMga/Wallet-Connect?node-id=13%3A3195

## How to test it
1) Open Wallet Connect app
2) Open any Dapp as Uniswap o Sushiswap in a new window and try to connect
3) When connecting => If the app exists as a safe app should prompt the user to open it if preferred
4) User can choose to continue using the Wallet Connect app
5) When executing transaction from the dapps the safe popup should appear

Check figma designs for the complete flow
